src/i18n/i18n.csv

```diff
@@ -77,6 +77,7 @@ Multimedia player,mmp-play,Play,Lire,Speel,Luaj,لعب,Խաղալ,Играя,播
 ,cc-off,Hide closed captioning,Masquer le sous-titrage,Steek Gesluit byskrifte,Hide Mbyllur captioning,أغلقت إخفاء السفلية,Թաքցնել փակ կատալոներեն,Скрий Затворени надписи,隱藏字幕,隐藏字幕,Skrýt Skryté titulky,Bijschrift verbergen,Peida subtiitrid,Ausblenden Untertitelung,Απόκρυψη Κλειστό λεζάντες,बंद कैप्शन छुपाएं,Feliratok kikapcsolása,Fela lokuðum skýringartextum,Sembunyikan subtitel,Nascondi i sottotitoli,ᐅᓂᒃᑳᑉ ᑕᐃᒎᓯᖏ ᓴᖅᑭᔮᕈᓐᓃᕐᓕ,クローズドキャプションを非表示,숨기기 캡션 휴무,Slēpt subtitrus,Slėpti didžiųjų raidžių įjungimą,Ukryj podpisy,Ocultar a legenda,Ocultar a legenda,Скрыть Субтитры,Skryť Skryté titulky,Ocultar subtítulos,ซ่อนปิด captioning,Kapalı eklentileri gizle,Сховати субтитри,Hide Đóng cửa phụ đề
 ,cc-on,Show closed captioning,Afficher le sous-titrage,Wys Gesluit byskrifte,Trego Mbyllur captioning,أغلقت تظهر السفلية,Ցուցադրել փակ կատալոներեն,Покажи Затворени надписи,顯示字幕,打开字幕,Zobrazit Skryté titulky,Bijschrift tonen,Näita subtiitreid,Zeige Untertitelung,Εμφάνιση Κλειστό λεζάντες,बंद कैप्शन दिखाएं,Feliratozás bekapcsolása,Sýna lokuðum skýringartextum,Tampilkan subtitel,Mostra i sottotitoli,ᐅᓂᒃᑳᑉ ᑕᐃᒎᓯᖏ ᓴᖅᑭᔮᕐᓕ,クローズドキャプションを表示,쇼는 캡션 휴무,Rādīt subtitrus,Rodyti didžiųjų raidžių įjungimą,Pokaż podpisy,Mostrar a legenda,Mostrar a legenda,Показать Субтитры,Zobraziť Skryté titulky,Mostrar subtítulos,แสดงปิด captioning,Kapalı eklentileri göster,Показати субтитри,Hiện Đóng cửa phụ đề
 ,cc-err,Error loading closed captions,Erreur dans le chargement des sous-titres,Fout tydens die laai gesluit onderskrifte,Captions Gabim gjatë ngarkimit të mbyllura,خطأ في تحميل تعليق مغلق,Սխալի բեռնելիս փակ վերնագրեր,Грешка при зареждане на затворени надписи,載入字幕錯誤,加载字母错误,Chyba při načítání uzavřené titulky,Fout bij het tonen van de bijschriften,Viga subtiitrite avamisel,Fehler beim Laden Untertiteln,Σφάλμα κατά τη φόρτωση κλειστά λεζάντες,लोड करने में त्रुटि बंद कैप्शन,Hiba a felirat betöltése közben,Villa við að hlaða Skjátextar,Galat saat memuat subtitel,Errore caricamento sottotitoli,ᑕᒻᒪᕐᓂᖅᑕᖃᖅᑐᖅ ᑕᐃᒎᓯᖏᓐᓂ ᓴᕿᔮᖅᑎᑦᓯᓂᕐᒥ,クローズドキャプションを実行できませんでした。,로드하는 중 오류가 발생 자막,Kļūda lejuplādējot subtitrus,Klaida kraunant didžiųjų raidžių įjungimą,Błąd w ładowaniu podpisów,Ocorreu um erro no carregamento da legenda,Ocorreu um erro no carregamento da legenda,Ошибка загрузки субтитров,Chyba pri načítaní uzavreté titulkami,Error al cargar subtítulos,ข้อผิดพลาดคำอธิบายปิดโหลด,Kapalı eklenti yükleme hatası,Помилка завантаження субтитрів,Lỗi khi đang nạp chú thích đóng
+,fs,Enter full screen,Ouvrir plein écran,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
Did you also added this in our master spreadsheet? Then you can run a special grunt command to update our local i18n.csv file.